pkg/sysinfo/numMem.go

```diff
@@ -0,0 +1,31 @@
+// +build amd64, arm, freebsd
```

Comment:
FreeBSD?

Reply:
We might just want to drop this, I don't think anything we ship runs on FreeBSD at present

Reply:
We also need a `numMem_unsupported` file that builds for windows, os x and returns just 0

Reply:
ok will make both of those changes now @mheon